code/online_challenges/src/project_euler/problem_005/problem_005.py

```diff
@@ -0,0 +1,17 @@
+def gcd(n1, n2):
```

Comment:
Also move all this code inside a function called `main`. Then use `if __name__ == '__main__'` and call `main` in it.